lib/graphql/schema/field/scope_extension.rb

```diff
@@ -4,7 +4,7 @@ module GraphQL
   class Schema
     class Field
       class ScopeExtension < GraphQL::Schema::FieldExtension
-        def after_resolve(value:, context:, **rest)
+        def after_resolve(object:, arguments:, context:, value:, memo:)
```

Comment:
Interestingly, `**rest` was allocating an object. Makes sense, but I never thought of it.